test/built-ins/Temporal/PlainDateTime/prototype/withCalendar/basic.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.withcalendar
+description: Non-throwing non-edge case
+features: [Temporal]
+includes: [temporalHelpers.js]
+---*/
+
+const dt = new Temporal.PlainDateTime(1976, 11, 18, 15, 23, 30, 123, 456, 789);
+const calendar = Temporal.Calendar.from("iso8601");
+
+TemporalHelpers.assertPlainDateTime(
+  dt.withCalendar(calendar),
+  1976, 11, "M11", 18, 15, 23, 30, 123, 456, 789,
+  "works"
+);
```

Comment:
Test that the result's calendar is actually `calendar`

Reply:
Good idea -- done.